conda/mpich/meta.yaml

```diff
@@ -37,6 +37,7 @@ requirements:
     - automake                  # [osx]
     - libtool                   # [osx]
     - make                      # [osx]
+    - {{ pin_compatible('openssl', lower_bound='1', upper_bound='3') }}
```

Comment:
OK, just making sure this doesn't pull version 3 - haven't looked up the logic here myself.

Reply:
I miss understood your comment... I thought you ment "you don't need the lower bounds"... 
But yes, upper_bounds is treated as less_than. Not less than or equal to.

Reply:
```yaml
requirements:
    build:
        - numpy {{ numpy }}
    run:
        - {{ pin_compatible('numpy', lower_bound='1.10', upper_bound='3.0') }}
```
This would yield a pinning of >=1.10,<3.0.


ref: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html